_includes/analytics.html

```diff
@@ -7,5 +7,8 @@
 
 ga('create', '{{ site.google_analytics }}', 'auto');
 ga('send', 'pageview');
+
+function clickview(idview){$(idview).bind('click',function(e){var b=$(this), type=b.attr('data-cv_active'), name=b.text(), h_from=window.location.href, h_to=false; if(b[0].nodeName=='A'){var h_to=b.attr('href');}; if(b.attr('data-cv_name')!==undefined){var name=b.attr('data-cv_name');}; ga('send', 'event', 'clickbutton',name, 'click_from:'+h_from+', click_to:'+h_to); if(h_to!==false){setTimeout(function(){if(b.attr('target')!==undefined){if(b.attr('target')!=='_blank'){window.open(h_to,'_self');}else{window.open(h_to);}}else{window.open(h_to,'_self');}},10); return false;};});};
```

Comment:
`data-cv_name`?

Reply:
for add custom name in tag for clickview function

Reply:
Yes, I get that, but the variable name is very strange. Let's just call it `ga-track-name` if so as that is much more descriptive of what it is.

Reply:
@vpetersson attribute rename